dmaws/commands/importdata.py

```diff
@@ -0,0 +1,31 @@
+import click
+
+from ..cli import cli_command
+from ..stacks import StackPlan
+from ..rds import RDSPostgresClient
+
+
+@cli_command('importdata', max_apps=0)
+@click.argument('export_file')
+def importdata_cmd(ctx, export_file):
+    """Import data into an environments database."""
+    plan = StackPlan.from_ctx(ctx, apps=['database_dev_access'])
+
+    if not plan.create(create_dependencies=False):
+        ctx.log("Failed to allow dev access", color='red')
+        sys.exit(1)
+
+    plan.info(['database'])
```

Comment:
I think we might be able to skip this, since creating dev access plan should gather database info and fill variable values anyway.